src/GraphNodeManagerViewExtension/ViewModels/GridNodeViewModel.cs

```diff
@@ -305,6 +306,24 @@ internal set
                 RaisePropertyChanged(nameof(InfoCount));
             }
         }
+        /// <summary>
+        /// Node returns null
+        /// </summary>
+        public bool IsRenamed
+        {
+            get
+            {
+                isRenamed = NodeModel.GetOriginalName() != NodeModel.Name;
```

Comment:
NodeModel doesn't already keep track of this?

Reply:
No, it doesn't, at least to what I could find. The NodeViewModel does, but we don't have access to it from an Extension. In any case, the NodeViewModel does exactly that - checks the OriginalName against the current NodeModel Name. It relies on the GetOriginalName() method, so I am replicating the workflow here.